conference.js

```diff
@@ -1442,6 +1442,10 @@ export default {
         room.on(ConferenceEvents.DISPLAY_NAME_CHANGED, (id, displayName) => {
             const formattedDisplayName
                 = displayName.substr(0, MAX_DISPLAY_NAME_LENGTH);
+            APP.store.dispatch(participantUpdated({
+                id,
+                name: formattedDisplayName
+            }));
             APP.API.notifyDisplayNameChanged(id, formattedDisplayName);
             APP.UI.changeDisplayName(id, formattedDisplayName);
```

Comment:
Good idea! 

But actually I did those changes by mistake. At the end it turned out that I don't need the display name in the store :)

I prefer to delete my display name changes than spending more time on this right now. Although  we should definitely do the change that you are proposing at some point.